Add `Allocatable` to `NodeStatus`

timstclair opened the pull request:
Adds `Allocatable` to `NodeStatus`, as described in
[17201](https://github.com/kubernetes/kubernetes/pull/17201)

Issue: https://github.com/kubernetes/kubernetes/issues/13984

dchen1107 commented:
Looks good to me overall. Please rebase it.

timstclair commented:
Done (the generated API files go out of date really quickly...).

dchen1107 commented:
--- /home/travis/gopath/src/github.com/kubernetes/kubernetes/api/swagger-spec/v1.json	2015-12-11 23:23:50.530006695 +0000
+++ /home/travis/gopath/src/github.com/kubernetes/kubernetes/_tmp/swagger-spec/v1.json	2015-12-11 23:18:24.000000000 +0000
@@ -13294,11 +13294,7 @@
     "properties": {
      "capacity": {
       "type": "any",
-      "description": "Capacity represents the total resources of a node. More info: http://releases.k8s.io/HEAD/docs/user-guide/persistent-volumes.md#capacity for more details."
-     },
-     "allocatable": {
-      "type": "any",
-      "description": "Allocatable represents the resources of a node that are available for scheduling. Defaults to Capacity."
+      "description": "Capacity represents the available resources of a node. More info: http://releases.k8s.io/HEAD/docs/user-guide/persistent-volumes.md#capacity for more details."
      },
      "phase": {
       "type": "string",
/home/travis/gopath/src/github.com/kubernetes/kubernetes/api/swagger-spec is out of date. Please run hack/update-swagger-spec.sh
!!! Error in hack/verify-swagger-spec.sh:28
  '"${KUBE_ROOT}/hack/after-build/verify-swagger-spec.sh" "$@"' exited with status 1
Call stack:

timstclair commented:
Ah, thanks. Reran `hack/update-all.sh`

dchen1107 commented:
lgtm

timstclair commented:
Rebased.

timstclair commented:
@k8s-bot test this

tallclair committed:
Add `Allocatable` to `NodeStatus`

Adds `Allocatable` to `NodeStatus`, as described in
[17201](https://github.com/kubernetes/kubernetes/pull/17201)

timstclair commented:
Fixed broken test: added Allocatable default to fuzzer.

timstclair commented:
@k8s-bot unit test this

timstclair commented:
Rebased.
